credmaster.py

```diff
@@ -58,13 +58,25 @@ def main(args,pargs):
 	if args.config != None:
 		log_entry("Loading AWS configuration details from file: {}".format(args.config))
 		aws_dict = json.loads(open(args.config).read())
-		access_key = aws_dict['access_key']
-		secret_access_key = aws_dict['secret_access_key']
-		profile_name = aws_dict['profile_name']
-		session_token = aws_dict['session_token']
-	if access_key is None and secret_access_key is None and session_token is None and profile_name is None:
-		log_entry("No FireProx access arguments settings configured, add access keys/session token or fill out config file")
-		return
+		access_key = aws_dict.get('access_key')
```

Comment:
The change to `.get()` just ensures we get `None` rather than a `KeyError` if the config file doesn't contain all the possibly fields.

Reply:
Config file parsing has changed, please remove this section as it is now handled elsewhere (see code, handled by function parse_all_args

The note about the `.get()` is good though, will implement that in the other config parsing